.github/workflows/contracts.yml

```diff
@@ -14,6 +14,7 @@ jobs:
     steps:
       - name: Checkout sources
         uses: actions/checkout@f25a3a9f25bd5f4c5d77189cab02ff357b5aedeb # v2.4.1
+        submodules: true
```

Comment:
Let's add this to all "Checkout sources" workflow steps, it's going to be a $noop for now while we don't use submodules.